Enable Carp in the Texinfo::Config namespace.

Fixes bulding media-video/ffmpeg-4.4 docs.

makeinfo: warning: error loading ./doc/t2h.pm: Undefined subroutine &Texinfo::Config::carp called at /usr/share/texinfo/Texinfo/Convert/HTML.pm line 7308.

diff -Nuar a/tp/texi2any.pl b/tp/texi2any.pl
--- a/tp/texi2any.pl	2021-06-30 16:25:37.000000000 +0200
+++ b/tp/texi2any.pl	2021-11-12 17:31:20.000000000 +0100
@@ -322,7 +322,7 @@
 {
 package Texinfo::Config;
 
-#use Carp;
+use Carp;
 
 # passed from main program
 my $cmdline_options;
